aws-sdk-s3-transfer-manager/README.md

```diff
@@ -0,0 +1,133 @@
+# AWS SDK S3 Transfer Manager
+
+A high performance Amazon S3 client for Rust.
```

Comment:
```suggestion
A high performance Amazon S3 transfer manager client for Rust.
```

We can also copy it from the https://github.com/awslabs/aws-c-s3 ReadMe and also copy the Key Features section.